actionpack/lib/abstract_controller/helpers.rb

```diff
@@ -84,10 +84,11 @@ def helper_method(*methods)
         file, line = location.path, location.lineno
 
         methods.each do |method|
+          # def current_user(*args, &block)
+          #   controller.send(:'current_user', *args, &block)
+          # end
           _helpers_for_modification.class_eval <<~ruby_eval, file, line
-            def #{method}(*args, &block)                    # def current_user(*args, &block)
-              controller.send(:'#{method}', *args, &block)  #   controller.send(:'current_user', *args, &block)
-            end                                             # end
+            def #{method}(*args, &block); controller.send(:'#{method}', *args, &block); end
```

Comment:
For simple cases like this you can do:

```ruby
class_eval <<~RUBY.lines.join(";"), ...
```

Reply:
Thanks, that's a bit neater. It doesn't work with the inline comments but it's still an improvement on what it was.